packages/react-native/ReactCommon/jsitooling/react/runtime/JSRuntimeFactoryCAPI.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ Shared interface that JSRuntimeFactory should implement.
+ */
```

Comment:
nit: the comment looks a little bit off. it's not an interface

Reply:
Changed